src/main/java/com/minecolonies/util/ServerUtils.java

```diff
@@ -86,19 +85,19 @@ public static EntityPlayer getPlayerFromUUID(@NotNull World world, @NotNull UUID
      * The {@link Permissions.Player} is a wrapper around a {@link UUID} of minecraft players.
      * The List will simply be converted into an {@link EntityPlayer} type.
      * <p>
-     * Uses {@link ServerUtils#getPlayerFromPermPlayer(Permissions.Player)}.
+     * Uses {@link ServerUtils#getPlayerFromPermPlayer(Permissions.Player, World)}.
      *
      * @param players The list of players to convert.
      * @return A list of {@link EntityPlayer}s
      */
     @NotNull
-    public static List<EntityPlayer> getPlayersFromPermPlayer(@NotNull List<Permissions.Player> players)
+    public static List<EntityPlayer> getPlayersFromPermPlayer(@NotNull List<Permissions.Player> players, @NotNull World world)

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): world [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)